hydroserving/client/contract.py

```diff
@@ -0,0 +1,374 @@
+from typing import Dict, Tuple
+import hydro_serving_grpc as hs_grpc
+import pandas as pd
+import yaml
+
+from google.protobuf.json_format import MessageToDict
+
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+
+from .type_conversions import HS_TO_NP_DTYPE, NP_TO_HS_DTYPE, NP_DTYPE_TO_ARG_NAME, STR_TO_HS_DTYPE, HS_DTYPE_TO_STR
+import numpy as np
+
+
+class ContractViolationException(Exception):
+    """
+    Exception raised after failed compliance check
+    """
+    pass
+
+
+class HSContract:
```

Comment:
I really don't like the idea of Yet Another Contract Object. 
Why not use the one provided in protos, and just write helper functions for it?